src/pages/Marketplace/Sidebar/Designations.tsx

```diff
@@ -6,10 +6,13 @@ import { FilterOption, FlatFilter } from "./common";
 const options: FilterOption<EndowDesignation>[] = [
   { displayText: "Non-Profit", key: "non-profit", value: "Non-Profit" },
   {
-    displayText: "Religious Non-Profit",
-    key: "religious-non-profit",
-    value: "Religious Non-Profit",
```

Comment:
Query:
i was told for most endowments the value is currently Non-profit or Religious Non-profit, will this change affect their filtering?